webapp/.storybook/preview.ts

```diff
@@ -36,40 +36,40 @@ const preview: Preview = {
         };
 
         // Fixing the theme for the DocsContainer
-        const el = document.querySelector("html");
-        const currentTheme = props?.context.store.globals.globals.theme;
-        el!.dataset['theme'] = currentTheme;
-        const theme = props?.context.store.globals.globals.theme === 'dark' ? themes.dark: themes.light;
+        const el = document.querySelector('html');
+        const currentTheme = (el?.dataset['colorMode'] as 'light' | 'dark') || 'light';
```

Comment:
Ah you mean  you fixed the storybook theme switcher not the app theme switcher, but the app theme switcher does not work as expected in storybook. Maybe that is also not too important